plugins/newsdownloader.koplugin/main.lua

```diff
@@ -15,7 +15,7 @@ local ltn12 = require("ltn12")
 local NewsDownloader = WidgetContainer:new{}
 
 local initialized = false  -- for only once lazy initialization
-local FEED_CONFIG_FILE = "feeds.xml"
+local FEED_CONFIG_FILE = "feed_config.lua"
```

Comment:
In terms of LUA vm, it just reads a stream of text and run the code, so extension doesn't matter. My concerns with `.conf` suffix are:
* inconsistency naming between other existing configs like `settings.reader.lua` and all the metadata in sdr folders.
* no indication of what config syntax is supported, i.e. when a user opens a `.conf` file, they won't be able to know what's the expected syntax.

Do you think it would good enough to have that `_config` part in the filename? In the long run, no one should be manually editing this file. The next iteration should add a UI for managing this config.